libethashseal/Ethash.h

```diff
@@ -45,7 +45,7 @@ class Ethash: public SealEngineBase
 
 	StringHashMap jsInfo(BlockHeader const& _bi) const override;
 	void verify(Strictness _s, BlockHeader const& _bi, BlockHeader const& _parent, bytesConstRef _block) const override;
-	void verifyTransaction(ImportRequirements::value _ir, TransactionBase const& _t, EnvInfo const& _env) const override;
+	void verifyTransaction(ImportRequirements::value _ir, TransactionBase const& _t, BlockHeader const& _header, u256 const& _startGasUsed) const override;
```

Comment:
The reason for this is that I find it weird that transaction validation depends on `EnvInfo` which is an enviroment info for VM execution.
Also `EnvInfo` now contains a pointer to `LastBlockHashesFace`, which shouldn't be needed by transation validation at all.